README.md

```diff
@@ -12,7 +14,7 @@ Add the dependency in your `build.gradle`:
 
 ```groovy
 dependencies {
-  implementation 'com.braintreepayments.api:drop-in:5.2.2'
+  implementation 'com.braintreepayments.api:drop-in:6.0.0-beta2'
```

Comment:
Do we plan on baking this as a beta for a bit, or does this need to be updated for release?

Reply:
This should automatically get updated with the latest version during our release process. The Rakefile update in this PR should fix the regex so that it correctly updates to 6.0.0 when we release GA.